File: 9.guessnumber.c
#include<stdio.h>  
#include<stdlib.h>
#include<stdbool.h>

int main()  
{
  int randomnumber = rand();
  int number;  
  bool correct = false;
  while(!correct){
    printf("Guess number: "); 
    scanf("%d",&number);  
    if(number>randomnumber){
      printf("Large\n"); 
    }
    else if(number<randomnumber){
      printf("Small\n"); 
    }
    else {
      correct=true;
      printf("You found correct number!"); 
    }
  }

}

File: 9.guessnumber.cs
using System;

class Program{
    static void Main(){
      Random rd = new Random();
      int randomnumber = rd.Next(1,100);
      int number = 0;
      bool correct = false;
      while(!correct){
        Console.WriteLine("Guess number: ");
        number = Convert.ToInt32(Console.ReadLine()); 
        if(number>randomnumber){
          Console.WriteLine("Large"); 
        }
        else if(number<randomnumber){
          Console.WriteLine("Small"); 
        }
        else {
          correct=true;
          Console.WriteLine("You found correct number!"); 
        }
      }
    }
}